.travis.yml

```diff
@@ -4,14 +4,16 @@ rvm:
   - 2.4
   - 2.5
   - 2.6
+  - 2.7
+  - ruby
```

Comment:
Does this just test with the latest official release of Ruby?  If so, you may want to add an `allow_failures` section for this too.  Although it's good to know if a new Ruby release breaks old code, you probably don't want that to fail the builds of incoming PRs.

Or possibly just remove this, and manually add new Ruby versions.  You probably want to explicitly list all supported versions to prevent gaps in coverage, anyway.

Reply:
Good call. I think I'll just remove it for now and I can add it later. What do you think about also removing `jruby`? Historically it's been a nightmare in `aws-sdk-ruby` with breaking tests. I wouldn't say that it's officially supported by us at this point.

Reply:
In theory, JRuby is great...

It's not officially supported by Rails either though.  Perhaps you can add an `allow_failures` section for it.  It looks like `aws-sdk-ruby` does this, and I've read it's helpful to people who do use JRuby, so they at least know the status of your library.

Reply:
Ah interesting. If rails doesn't officially support it - I think it's okay to make it unofficially supported here too; after all, this is supposed to be a rails plugin package. Though I would expect JRuby users wouldn't run into any issues using this anyway. For now, I'll keep jruby and add an allowed failure if it becomes a problem.